### PR TITLE
 -auto-approve

### PR DESCRIPTION
**Pull request by biofool**
it's mandatory CLI: v0.4.2

### CODE REVIEW

File: website/content/docs/getting-started.mdx [hunk 1]
```diff
@@ -177,8 +177,7 @@ waypoint ui -authenticate
 When you are done, run `destroy` to remove the application from Docker.
 
 ```shell-session
```
Comment (briancain) [inline website/content/docs/getting-started.mdx:179]:
```suggestion
When you are done, run `destroy` to remove the application from Docker.
Notice that the `destroy` command requires confirmation to run.

```shell-session
```

File: website/content/docs/getting-started.mdx [hunk 2]
```diff
@@ -177,8 +177,7 @@ waypoint ui -authenticate
...
-$ waypoint destroy
-```
+$ waypoint destroy  -auto-approve```
```
Comment (briancain) [inline website/content/docs/getting-started.mdx:180]:
```suggestion
$ waypoint destroy  -auto-approve
```
```

**Review by briancain — CHANGES_REQUESTED** [review]
Hey there! Thanks for catching this in our docs. I've left a few PR fixes that should go along with it. Also if you could, please sign our CLA. Thank you!

**Comment by biofool** [comment]
Glad to help out.

File: website/content/docs/getting-started.mdx [hunk 3]
```diff
@@ -175,10 +175,10 @@ waypoint ui -authenticate
 ## Destroy
 
 When you are done, run `destroy` to remove the application from Docker.
+Notice that the `destroy` command requires confirmation to run.
 
 ```shell-session
-$ waypoint destroy
-```
+$ waypoint destroy  -auto-approve
```
Comment (briancain) [inline website/content/docs/getting-started.mdx:181]:
```suggestion
$ waypoint destroy -auto-approve
```

**Review by briancain — APPROVED** [review]

**Pull request merged**